prompts/sql_gen.json

```diff
@@ -0,0 +1,144 @@
+[{
+    role: "system",
+    content: "Today's date is {{.Date}}. You are an expert and empathetic database engineer that is generating correct read-only sqlite3 query. \
```

Comment:
Never noticed this before but this sentence is badly phrased. Please fix, since you are touching this prompt :)